src/main/java/uk/gov/pay/api/model/CardPayment.java

```diff
@@ -39,14 +39,19 @@
     @JsonProperty("total_amount")
     private final Long totalAmount;
 
+    @JsonProperty("provider_id")
+    private final String providerId;
```

Comment:
Can we leave it as "string"? Problem is, each payment provider has their own format when providing an id. Epdq uses a 10 digit numeric format, Worldpay uses UUIDs for example.

Reply:
"string" may looks ugly along with all other field examples. We can add it similar to other fields examples we provide(`"reference": "your-reference",`). In this case it could be `reference-from-payment-gateway` / or something similar

Reply:
ok, i've added `@ApiModelProperty(example = "reference-from-payment-gateway")`